bridge/client/app/_components/ktb-trigger-sequence/ktb-trigger-sequence.component.ts

```diff
@@ -199,18 +199,14 @@ export class KtbTriggerSequenceComponent implements OnInit, OnDestroy, AfterView
   }
 
   public setTimeframe(timeframe: Timeframe): void {
-    if (!this.isTimeframeEmpty(timeframe)) {
-      this.isValidTimeframe =
-        (timeframe.hours ?? 0) * 60 +
+    this.isValidTimeframe = this.isTimeframeEmpty(timeframe)
+      ? true
+      : (timeframe.hours ?? 0) * 60 +
           (timeframe.minutes ?? 0) +
           (timeframe.seconds ?? 0) / 60 +
           (timeframe.millis ?? 0) / 60_000 +
           (timeframe.micros ?? 0) / 60_000_000 >=
```

Comment:
Is this still necessary?

Reply:
When it is still there, we could enable millis and micros again (if it is necessary).

Reply:
I'd rather go for removing it, as we can also introduce it if needed later on. YAGNI principle applied, and less code is less maintenance :)

Reply:
Changed